forms-flow-theme/scss/_modal.scss

```diff
@@ -551,3 +506,49 @@
     align-self: stretch;
   }
 }
+
+//reusable multi-select
+.multi-select{
+  .list-group {
+    border: 1px solid var(--ff-base-600);
+    border-radius: var(--radius-0100);
+  
+    .list-group-item {
+      border: none;
+    }
+  }
+  .input-drop {
+    position: absolute;
+    z-index: 100;
+    overflow: auto;
+    border-radius: var(--radius-0100);
+    border-bottom: 1px solid var(--base-base-600);
+  }
+  
+  .input-with-pills {
+    display: flex !important;
+    justify-content: flex-start;
+    flex-wrap: wrap !important;
+    align-items: center;
+    padding: var(--spacer-050) var(--spacer-100);
+    border: 1px solid var(--ff-base-600) !important;
+    border-radius: var(--radius-0100) !important;
+    position: relative;
+    grid-row-gap: 5px !important;
+    row-gap: 5px !important;
+    grid-column-gap: 5px !important;
+    column-gap: 5px !important;
```

Comment:
we can use gap: 5px !important instead of this 4 lines. pls cross check once

Reply:
Updated